src/terms/ewald.jl

```diff
@@ -9,28 +9,23 @@ struct Ewald end
 (::Ewald)(basis) = TermEwald(basis)
 
 struct TermEwald{T} <: Term
-    energy::T  # precomputed energy
+    energy::T                # precomputed energy
+    forces::Vector{Vec3{T}}  # and forces
 end
-function TermEwald(basis::PlaneWaveBasis{T}) where {T}
-    TermEwald(T(energy_ewald(basis.model)))
+@timing "precomp: Ewald" function TermEwald(basis::PlaneWaveBasis{T}) where {T}
+    energy, forces = energy_forces_ewald(basis.model; compute_forces=true)
```

Comment:
If we anyway always precompute them. Why don't we always compute them? Would make things even simpler.

Reply:
Also you can now use
```
(; energy, forces) = 
```
as we moved to at least Julia 1.7.